src/dmd/expression.d

```diff
@@ -5152,6 +5152,7 @@ extern (C++) final class ImportExp : UnaExp
  */
 extern (C++) final class AssertExp : UnaExp
 {
+    // https://dlang.org/spec/expression.html#assert_expressions
```

Comment:
Why don't you add this in the Ddoc header of `AssertExp`?

Reply:
ok